### DISCUSSION TITLE
Modify data storage approach

### DISCUSSION THREAD

**Discussion by ShunyangLi** [JanusGraph/janusgraph#2405] [Q&A]
Hi, I read the Docs about janusgraph. I am considering is that possible to add new storage backend? For example, add Redis as the storage. Is that hard?

**Comment by li-boxuan** [comment]
Yes, it is totally possible to add a new storage backend.

You can start by reading some relevant sources:

https://groups.google.com/g/janusgraph-dev/c/9JrMYF_01Cc/m/UfIGcD98AAAJ 
https://www.youtube.com/watch?v=rQM_ZPZy8Ck
https://github.com/Playtika/aerospike-janusgraph-storage-backend which is also an in-memory distributed key-value storage
